integrated_channels/moodle/transmitters/content_metadata.py

```diff
@@ -2,9 +2,19 @@
 """
 Class for transmitting content metadata to Moodle.
 """
+import logging
+from itertools import islice
 
+from django.conf import settings
+
+from integrated_channels.exceptions import ClientError
 from integrated_channels.integrated_channel.transmitters.content_metadata import ContentMetadataTransmitter
 from integrated_channels.moodle.client import MoodleAPIClient
+from integrated_channels.utils import chunks
+
+LOW_CHUNK_SIZE = 5
```

Comment:
A little arbitrary, but was able to transmit 5 courses with the full 2000 character limit long description successfully.

I thought of just doing a division each attempt, but that might be taxing performance-wise and I didn't feel like utilizing rounding to make sure I could divide by 2 (or whatever digit, but by half each time seemed reasonable)